docs/cep/sink/sink-types/http-call.md

```diff
@@ -10,87 +10,87 @@ The http-call sink publishes messages to endpoints via HTTP or HTTPS protocols u
 
 ## Query Parameters
 
-| Name                            | Description      | Default Value    | Possible Data Types | Optional | Dynamic |
-|---------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|---------------------|----------|---------|
-| publisher.url                   | The URL which should be called. Examples: `http://localhost:8080/endpoint`, `https://localhost:8080/endpoint`                                        |                  | STRING              | No       | No      |
-| sink.id                         | Identifier to correlate the http-call sink to its corresponding http-call-response sources to retrieved the responses.                               |                  | STRING              | No       | No      |
+| Name           | Description      | Default Value    | Possible Data Types | Optional | Dynamic |
+|--------------|------------------------------|-------------------------|---------------------|----------|---------|
+| publisher.url  | The URL which should be called. Examples: `http://localhost:8080/endpoint`, `https://localhost:8080/endpoint`      | | STRING              | No       | No      |
```

Comment:
The `http://localhost:8080/endpoint`, `https://localhost:8080/endpoint` URLs are invalid for us. Please take them out.

Reply:
Should I replace them or just remove them?